tests/bcl-test/BCLTests/BCLTests-mac.csproj.in

```diff
@@ -51,11 +51,11 @@
     <PackageReference Include="System.Buffers" Version="4.5.0" />
     <PackageReference Include="System.Memory" Version="4.5.2" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
-    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.analyzers" Version="0.10.0" />
-    <PackageReference Include="xunit.extensibility.core" Version="2.4.1" />
-    <PackageReference Include="xunit.extensibility.execution" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.utility" Version="2.4.1" />
+    <PackageReference Include="xunit.extensibility.core" Version="2.4.0" />
+    <PackageReference Include="xunit.extensibility.execution" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.utility" Version="2.4.0" />
```

Comment:
any reason for the downgrade ?
@mandel-macaque was 2.4.1 used for some specific reason ?

Reply:
I just made it consistent with the other test projects which all use 2.4.0, there was no particular reason. I can revert it back but it seems to make no difference. We should probably refactor the version we use into a common .props file for all projects.